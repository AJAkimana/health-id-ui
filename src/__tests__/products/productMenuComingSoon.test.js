import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { shallow, mount } from 'enzyme';
import { MockedProvider } from 'react-apollo/test-utils';
import MockComponent from '../../../__mocks__/mockComponent';
import toJson from 'enzyme-to-json';
import ProductMenuComingSoon from '../../components/products/productMenuComingSoon';

jest.mock('../../components/products/productMenuComingSoon', () => MockComponent);

const props = {
  variables: {},
  classes: {},
  session: {
    me: {
      role: {
        name: 'Master Admin'
      },
      activeOutlet: {
        outletpreference: {
          outletTimezone: {
            name: "Rwanda/Kigali"
          }
        }
      }
    }
  }
};

describe('ProductMenuComingSoon', () => {
  it('renders without crashing', async() => {
    const wrapper = mount(
      <MockedProvider addTypeName={false}>
        <BrowserRouter>
          <ProductMenuComingSoon {...props} />
        </BrowserRouter>
      </MockedProvider>
    );
  
    expect(wrapper).toHaveLength(1);
  });

  it('should render the ProductMenuComingSoon component properly', () => {
      const wrapper = shallow(<ProductMenuComingSoon {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot();
  })
});
