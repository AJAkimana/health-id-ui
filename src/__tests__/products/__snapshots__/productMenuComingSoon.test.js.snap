// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProductMenuComingSoon should render the ProductMenuComingSoon component properly 1`] = `<div />`;
